plugins/deploy_env_vars/lib/samson_deploy_env_vars/samson_plugin.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module SamsonDeployEnvVars
+  class Engine < Rails::Engine
+    config.after_initialize do
+      # Decorates default deploys helper. Maybe there is a better way to do this?
+      require_relative '../../app/helpers/deploys_helper'
```

Comment:
the regular `inherited` hack won't work since they don't inherit anything
I'd recommend using a `:decorate_helper` hook to make that work ... so the original `deploys_helper.rb` gets a `.fire :decorate_helper, :deploys_helper` for example and here we'd add a callback to trigger the require

Reply:
I see. Will investigate tomorrow how can I do that (I guess that in a similar way as controllers and models are already doing)